### PR TITLE
[K12] Add task to K12 yml to support New Picklists Value for Citizenship Status on Contact

### LINKED ISSUE #295: [K12] Add task to K12 yml to support New Picklists Value for Citizenship Status on Contact
**Note: FOR INTERNAL USE ONLY. For support questions, please use the EDA Chatter group in [The Power of Us Hub](https://powerofus.force.com).
For feature requests, please use the Ideas tab in [The Power of Us Hub](https://powerofus.force.com).**

* **Do you want to request a *feature* or report a *bug*?**
Feature
* **What is the current behavior?**
Currently, hed__Citizenship_Status__c do not have a dropdown option for non US resident. 
* **What is the expected behavior?**
hed__Citizenship_Status__c should contain a picklist value that supports non US resident. 
* **What is the motivation / use case for changing the behavior?**
Allows for non US resident to identify their status.

### PR DESCRIPTION
**Pull request by kdy618**
# Critical Changes

# Changes
Added a new task to K12 cumulus.yml file to reference an EDA task that adds new picklist value for hed__Citizenship_Status__c on Contact. 
# Issues Closed
Close #295
# New Metadata

## Unpackaged Metadata

# Deleted Metadata

# Testing Notes

### CODE REVIEW

File: cumulusci.yml
Comment (MatthewBlanski) [inline cumulusci.yml:190]: This is a task calling a task, which will not function in CumulusCI. Tasks call Python Classes.
```suggestion
```

File: cumulusci.yml
Comment (MatthewBlanski) [inline cumulusci.yml:550]: This will accomplish calling the task from EDA defined as eda in this yml file.
```suggestion
                task: eda:add_citizenship_status_values
```

**Review by MatthewBlanski — CHANGES_REQUESTED** [review]

**Review by MatthewBlanski — DISMISSED** [review]
Tentatively approved. This functioned when I changed over to the eda beta, which has the citizenship picklist task. We will need to wait until the release merge (which is appropriate) until we merge this in.

@spinkelman FYI
@jofsky FYI

**Comment by jofsky** [comment]
@kdy618 I've discussed my idea of having an upstream EDA configuration task to be passed downstream to allow all the products to reduce code duplication. I'll approve this for now but lets make this part of our larger effort to reduce this code footprint.

File: cumulusci.yml
Comment (jofsky) [inline cumulusci.yml:26]:
```suggestion
tasks:
```

**Review by jofsky — DISMISSED** [review]
@kdy618 I've discussed my idea of having an upstream EDA configuration task to be passed downstream to allow all the products to reduce code duplication. I'll approve this for now but lets make this part of our larger effort to reduce this code footprint.

**Comment by kdy618** [comment]
@MatthewBlanski  @jofsky , I had to fix merge conflict. If you don't mind re-reviewing this PR. Thank you in advance.

**Review by MatthewBlanski — APPROVED** [review]

**Comment by jofsky** [comment]
@kdy618 what's the status of this?

**Comment by kdy618** [comment]
@jofsky, I initially was waiting for approval and based on Matt's initial response https://github.com/SalesforceFoundation/k12-architecture-kit/pull/296#pullrequestreview-553953949 as to why this PR was not merged. I had to fix merge conflict when the cumulus file was updated and resubmitted for approval. It looks like @MatthewBlanski  approved it and was waiting for your approval. However, it looks like it hit another merged conflict. Is there a better way for me to get this PR closed?

**Review by jofsky — DISMISSED** [review]
@kdy618 we should probably run prettier to fix the linting failure but this looks fine.

**Comment by kdy618** [comment]
@jofsky @MatthewBlanski , fixed the linter :D

**Review by jofsky — APPROVED** [review]
Great work!

**Review by MatthewBlanski — APPROVED** [review]
Happy path testing resolved successfully. The code and references look to be in order.

**Comment by spinkelman** [comment]
QA Passes - Ran trial > trial_org - Verified picklist values appear in org and no error on task run in cci.
Waiting to merge until after release merge per Matt's comment above.

**Pull request merged**
